css/build.css

```diff
@@ -267,6 +267,11 @@ input.form-control {
     max-width: 100%;
   }
 
+  .btn-primary .update-password {
+    min-width: 30%;
```

Comment:
As mentionedin https://github.com/Fliplet/fliplet-studio/issues/4924#issuecomment-566984656 this isn't just affecting one button, but all buttons in this component.

1. The `.btn-primary` and `.btn-secondary` selectors in this file should be updated so that they only apply to the buttons within this component instead of all buttons on the page.
1. The `min-width` fix should apply to all affected buttons, not just the update password button.

Reply:
fixed